services/ui-src/src/components/reports/DynamicModalOverlayReportPage.tsx

```diff
@@ -194,15 +194,23 @@ const sx = {
       paddingRight: "0",
       borderBottom: "1px solid",
       borderColor: "palette.gray_light",
-      ".tablet &, .mobile &": {
-        border: "none",
-      },
       "&:nth-of-type(1)": {
         width: "2.5rem",
       },
       "&:nth-of-type(3)": {
         width: "260px",
       },
     },
+    /**
+     * removes bottom border only for
+     * State or Territory-Specific Initiatives
+     */
+    tr: {
+      "&:last-of-type": {
+        td: {
+          border: "none",
```

Comment:
Honestly can't tell where to look for this change. Could you add a screenshot for posterity?

Reply:
Sure! This is what it looked like before, with that border on the last row sitting adjacent to the footer border:

<img width="886" alt="Screenshot 2024-09-27 at 2 02 07 PM" src="https://github.com/user-attachments/assets/ad810c63-5286-495f-89ac-0ffa21814aac">